packages/gatsby/src/utils/webpack-modify-validate.js

```diff
@@ -35,5 +48,12 @@ export default (async function ValidateWebpackConfig(config, stage) {
     console.log(`\n`)
   })
 
+  console.log(
+    `Your Webpack config does not appear to be valid. This could be because of
+something you added or a plugin. If you don't recognize the invalid keys listed
+above try removing plugins and rebuilding to identify the culprit.
```

Comment:
This looks sort of ugly. Open to suggestions as to how to format multi-line strings so they look nice in code.